full-stack-tests/core/src/frontend/standalone/ITwinError.test.ts

```diff
@@ -0,0 +1,73 @@
+import { BentleyError, IModelHubStatus, ProcessDetector } from "@itwin/core-bentley";
+import { BackendError, ChannelError, ConflictingLock, ConflictingLocksError, LockState } from "@itwin/core-common";
+import { expect } from "chai";
+import { coreFullStackTestIpc } from "../Editing";
+import { TestUtility } from "../TestUtility";
+
+if (ProcessDetector.isElectronAppFrontend) {
+
+  describe("ITwinError exceptions from backend", async () => {
```

Comment:
```suggestion
  describe("ITwinError exceptions from backend  (#integration)", async () => {
```

Reply:
Ah good catch, just saw the original test file had the integration test; not applicable here. closing